test/tests/geomsearch/3d_penetration_locator/gold/3d_rings_out_values_0001.csv

```diff
@@ -105,7 +105,7 @@ id,penet,x,y,z
 103,0,0.023321254371987,0.085147669460424,0.52757575757576
 104,0,0.01343582997789,0.080003415385772,0.52757575757576
 105,0,0.001228247892379,0.074916002753794,0.52757575757576
-106,-0.02,-0.012040599039569,0.069254560506297,0.52757575757576
+106,0,-0.012040599039569,0.069254560506297,0.52757575757576
```

Comment:
I assume ids 106 and 146 were being erroneously picked up?

Reply:
![image](https://github.com/user-attachments/assets/1c168052-7af9-452e-8e56-51d8f9f5eb3e)
Yup, blue is in contact